nbconvert/exporters/html.py

```diff
@@ -61,6 +65,17 @@ def default_config(self):
         c.merge(super(HTMLExporter,self).default_config)
         return c
 
+    def markdown2html(self, source):
+        """Markdown to HTML filter respecting the anchor_link_text setting"""
+        renderer = IPythonRenderer(escape=False,
+                                   anchor_link_text=self.anchor_link_text)
+        return MarkdownWithMath(renderer=renderer).render(source)
+
+    def default_filters(self):
+        for pair in super(HTMLExporter, self).default_filters():
+            yield pair
+        yield ('markdown2html', self.markdown2html)
```

Comment:
@michaelpacer I think this is the main difference from #508: rather than passing the new option through the template to the existing filter, I've replaced the `markdown2html` filter with one that follows the config option.

Reply:
@takluyver Hello. I am still struggling with getting rid of Pilcrow sign from Markdown Headings in Jupyter NB. I have followed up on your suggested solution and all conversations related to this issue. I accessed the html.py (Exporters folder) and strings.py and markdown_mistune.py (Filters folder) under nbconvert, and attempted a hack by replacing the Unicode for Pilcrow with the Unicode for space ' ' = u'\u0020'. I saved the files, then I tried to execute the following code snippets in my main Jupyter Notebook

`import traitlets
from traitlets.config import Config
c=Config()
c.HTMLExporter.anchor_link_text=u'\0020' 

import nbconvert
import mistune
nbconvert.filters.markdown_mistune.IPythonRenderer.header=mistune.Renderer.header`

However, there seems to be no change and the Pilcrows are still crowding against markdown headings in my Notebook.

Your solution seems to be the only suggestion available online for this issue, urge you to kindly help me with any bright ideas on this. I desperately need to get rid of that symbol, then my nbconvert pdf conversion will be successful and presentable.

Thanks.